src/components/AsideHeader/components/CompositeBar/CompositeBar.module.scss

```
@@ -26,74 +41,75 @@ $block: '.#{variables.$ns}composite-bar';
-        min-height: #{variables.$item-height};
+        min-height: var(--gn-aside-header-item-height, #{variables.$item-height});
     }
 
     &__menu-group-nested-item[class] {
         position: relative;
-        padding-inline-start: var(--g-spacing-3);
+
+        #{$block}__menu-group-nested-connector {
+            position: absolute;
+            inset-inline-start: calc(
+                var(--gn-aside-header-item-height, #{variables.$item-height}) / 2 -
```

Stylelint (17.13.0)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)